src/scenarios/scenarios_4.ts

```diff
@@ -1,16 +1,45 @@
-import {SubstrateService} from "../services/substrate-service";
-import {ScenarioInterface} from "./scenario-interface";
-import Logger from './../services/logger';
+import { SubstrateService } from "../services/substrate-service";
+import { ScenarioInterface } from "./scenario-interface";
+import Logger from "./../services/logger";
 
 export class Scenarios_4 implements ScenarioInterface {
   logger = new Logger(Scenarios_4.name);
 
   async run() {
-    /*
-       TODO: Implement Scenario 4 [Restricted Assets]:
-         * Create 3 user wallets
-         * Sends Restricted Assets to those wallets using issueRestrictedAsset function
-     */
-    this.logger.log('Scenario 4');
+    this.logger.log("Creating SubstrateService instance scenario 4...");
+
+    const service = new SubstrateService();
+    await service.initialize();
+
+    this.logger.log(
+      `SubstrateService instance has been created and initialized successfully`
+    );
+
+    // Generating Wallets
```

Comment:
No need to add comment, since we have the same text in the logging below

Reply:
Removed the comment